src/Popper.js

```diff
@@ -145,11 +149,11 @@ export class InnerPopper extends React.Component<PopperProps, PopperState> {
 
     if (!referenceElement || !popperNode) return;
 
-    this.popperInstance = new PopperJS(
+    this.popperInstance = (new PopperJS(
       referenceElement,
       popperNode,
       this.getOptions()
-    );
+    ): any);
```

Comment:
why is this necessary? outdated types?

Reply:
If that's the case we really need to fix the types, otherwise we are missing most of the type coverage with this change.

Reply:
I added the PR upstream already

Reply:
I merged and released it

Reply:
Nice! I'll update the PR, thanks!